plugins/org.python.pydev.parser/src/org/python/pydev/parser/visitors/NodeUtils.java

```diff
@@ -2174,4 +2175,80 @@ public static exprType extractOptionalValueSubscript(exprType node) {
         return node;
     }
 
+    public static List<String> extractValuesFromSubscriptSlice(sliceType slice) {
+        List<String> values = new ArrayList<String>();
+
+        if (slice instanceof ExtSlice) {
+            ExtSlice extSlice = (ExtSlice) slice;
+            if (extSlice != null && extSlice.dims != null) {
+                for (sliceType dim : extSlice.dims) {
+                    if (dim instanceof Index) {
```

Comment:
The code right below in this block does deal with an `Index` too without resorting directly to NodeUtils.getRepresentationString -- shouldn't that code be refactored for the `Index` type and both this place and that one use the same code?